src/main/java/com/mael/payroll/paymentTypes/HourlyPayment.java

```diff
@@ -38,7 +38,7 @@ public TimeCard getTimeCard(LocalDate date) {
     public double calculatePay(LocalDate payDay) {
         double pay = 0.0;
         for(TimeCard timeCard : timeCards) {
-            if (inCurrenWeek(payDay, timeCard)) {
+            if (isInPeriod(payDay, timeCard)) {
```

Comment:
Yeah, I was too fast maybe. 

This is something I wanted to do, have all the methods in the different payment types that calculate the period in one method with the same name, so that it was more obvious that the method doesn't belong there and should be moved to the schedules. Since the schedules are named after the periods they last for, it would be obvious then. See my comment on #8. I probably shouldn't have started doing it if I was not going to move the method yet :sweat: .